src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -317,6 +317,46 @@ public function symlink($originDir, $targetDir, $copyOnWindows = false)
     }
 
     /**
+     * Return the target of a link.
+     *
+     * @param string $path        A link path.
+     * @param bool   $finalTarget Whether or not to recursively follow links
+     *                            until a final target is reached.
+     *
+     * @return string Return the resolved link.
+     *
+     * @throws IOException When the link does not exist or is not readable.
+     */
+    public function readlink($path, $finalTarget = false)
```

Comment:
I did the change, but I think the name "realpath" is misleading. The PHP realpath do more things than resolving links (it also canonicalizes and normalizes the path: http://php.net/manual/fr/function.realpath.php). I'm not sure what happens in our case on Windows / Unix and if the behavior is really the same everywhere. Perhaps should we find a better name.